README.md

```diff
@@ -26,7 +38,213 @@ required for testing citus.
 * [Advanced fab Usage](#advanced-fab)
   * [Using Multiple Citus Installations](#multiple-installs)
 
-# <a name="getting-started"></a> Getting Started
+## <a name="azure"></a>Azure
+
+## <a name="azure-getting-started"></a> Getting Started
+
+You can find more information about every step below in other categories. This list of commands show how to get started quickly. Please see other items below to understand details and solve any problems you face.
+
+You should have `az cli` in your local to continue.
+
+You should use `ssh-agent` to add your ssh keys, which will be used for downloading the enterprise repository. Note that your keys are kept only in memory, therefore this is a secure step.
+
+In `azuredeploy.parameters.json` file, you will see the parameters that you can change. For example if you want to change the number of instances, you will need to change the parameter `numberOfInstances`. You can change the type of coordinator and workers separately from the parameters file. Also by default for workers, memory intense vms are used(E type) while for coordinator CPU intense vms are used(D type).
+
+After you run tests, you can see the results in `results` folder. The `results` folder will have the name of the config used for the test.
+
+## <a name="azure-setup-steps"></a> Setup Steps For Each Test
+
+You will need to follow these steps to create a cluster and connect to it, on your local machine:
+
+```bash
+
+# start ssh agent
+eval `ssh-agent -s`
+
+# Add your Github ssh key for enterprise (private) repo
+ssh-add
+
+# Put your public key to sshPublicKey in azuredeploy.parameters.json. You can see your public key with:
+cat ~/.ssh/id_rsa.pub
+
+# in the session that you will use to ssh, set the resource group name
+export RESOURCE_GROUP_NAME=talha_test_resource_group
+
+# Quickly start a cluster of with defaults. This will create a resource group and use it for the cluster.
+./create-cluster.sh
```

Comment:
Hmm, I'm currently blocked. I get the following error:

```
azure/create-cluster.sh
{
  "id": "/subscriptions/7fec3109-5b78-4a24-b834-5d47d63e2596/resourceGroups/onder_test_auto",
  "location": "eastus2",
  "managedBy": null,
  "name": "onder_test_auto",
  "properties": {
    "provisioningState": "Succeeded"
  },
  "tags": null,
  "type": "Microsoft.Resources/resourceGroups"
}
[Errno 2] No such file or directory: 'azuredeploy.json'
```

Reply:
run this in the folder `azure`

Reply:
this is because `azuredeploy.json` is a relative file path, not absolute file path. We can use an absolute path here later.